Revert https://github.com/rust-lang/git2-rs/commit/b830b24c8f70
until devel/libgit2 is updated to 1.1.0

warning: libgit2/src/transports/ssh.c:11:10: fatal error: 'libssh2.h' file not found
warning: #include <libssh2.h>
warning:          ^~~~~~~~~~~
warning: 1 error generated.

error: failed to run custom build command for `libgit2-sys v0.12.15+1.1.0`

--- cargo-crates/libgit2-sys-0.12.15+1.1.0/build.rs.orig	2020-12-18 15:47:36 UTC
+++ cargo-crates/libgit2-sys-0.12.15+1.1.0/build.rs
@@ -12,7 +12,7 @@ fn main() {
     // To use zlib-ng in zlib-compat mode, we have to build libgit2 ourselves.
     if !zlib_ng_compat {
         let mut cfg = pkg_config::Config::new();
-        if let Ok(lib) = cfg.atleast_version("1.1.0").probe("libgit2") {
+        if let Ok(lib) = cfg.atleast_version("1.0.1").probe("libgit2") {
             for include in &lib.include_paths {
                 println!("cargo:root={}", include.display());
             }
